### PR TITLE
CB-13310 - Blog post for cordova-ios@4.5.1

### PR DESCRIPTION
**Pull request by surajpindoria**
### Platforms affected
iOS

### What does this PR do?
Blog post for cordova-ios@4.5.1 release

### What testing has been done on this change?
Previewed the website/post locally

### Checklist
- [x] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [x] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

**Comment by surajpindoria** [comment]
Forgot to fill in template, updated.

**Comment by shazron** [comment]
Please add links to the other iOS 11 issues that have workarounds in CSS etc that have been discussed. cc @dpogue what relevant blog posts need to be highlighted?

**Comment by dpogue** [comment]
Note that in order to take advantage of the iPhone X screen size, you must use Launch Storyboards. Splash Screen images will result in letterboxing (so existing apps are safe).

Viewport-related blog posts:
* The official WebKit one: https://webkit.org/blog/7929/designing-websites-for-iphone-x/
* My post focusing more specifically on Cordova apps: https://ayogo.com/blog/ios11-viewport/
* Stephen's post on Safari in general (not Cordova-specific): http://stephenradford.me/removing-the-white-bars-in-safari-on-iphone-x/

Maybe add a note that the Status Bar plugin is currently being worked on to resolve these issues?

**Comment by surajpindoria** [comment]
Just updated the post based on your feedback @shazron @dpogue

File: www/_posts/2017-09-25-ios-release.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+tags: news releases
+---
+
+We are happy to announce a minor version of `Cordova iOS 4.5.1` has been released!
+
+This version provides updates for the latest iOS 11 and Xcode 9. You can now create builds for this new version and properly deploy to either the emulator or device.
+
+Things to note:
+
+* iPhone X new screen size, must use Launch Storyboards to take full advantage
```
Comment (shazron) [inline www/_posts/2017-09-25-ios-release.md]: grammar: For iPhone X's new screen size, you must use Launch Storyboards to take full advantage of it

File: www/_posts/2017-09-25-ios-release.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* Current Splash Screen images will result in letterboxing
+
+Apple has also made some changes that could effect your current apps, specifically around the viewport. Here are a few resources that could help with your transition to iOS 11:
+
+1. https://webkit.org/blog/7929/designing-websites-for-iphone-x/
+2. https://ayogo.com/blog/ios11-viewport/
+3. http://stephenradford.me/removing-the-white-bars-in-safari-on-iphone-x/
```
Comment (shazron) [inline www/_posts/2017-09-25-ios-release.md]: Please wrap these in the proper Markdown syntax for links

File: www/_posts/2017-09-25-ios-release.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+We are also aware that there are specific issues related to the Status Bar plugin. The team is working resolve these for the next release of the plugin.
```
Comment (shazron) [inline www/_posts/2017-09-25-ios-release.md]: grammar: The team is working to resolve these for the next release of the plugin.

**Comment by audreyso** [comment]
Merged and deployed! Please close :) @surajpindoria

**Pull request closed without merging**